integration/kubernetes/init.sh

```diff
@@ -18,11 +18,13 @@ source "${SCRIPT_PATH}/../../.ci/lib.sh"
 sudo iptables -P FORWARD ACCEPT
 
 # Remove existing CNI configurations:
-cni_interface="cni0"
 sudo rm -rf /var/lib/cni/networks/*
 sudo rm -rf /etc/cni/net.d/*
-sudo ip link set dev "$cni_interface" down
-sudo ip link del "$cni_interface"
+cni_interface="cni0"
+if ip a | grep -q "$cni_interface"; then
```

Comment:
As `$cni_interface` is never null, you can probably optimise this with something like `if [ ip a show $cni_interface ]`.

Reply:
liked the idea, made the change.